MovieApp/DataLayer/Clients/BaseApiClient.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+
+class BaseApiClient {
+
+    private let baseUrl: String
+    let apiKey = "e24dd8d2f3822e3917d10c6570d7f574"
```

Comment:
can this be private?
also: fields initialized on declaration before fields not init on declaration